doc/bitmasks.rst

```diff
@@ -2,4 +2,71 @@
 Bit Masks in DESI
 =================
 
-This page describest bit masks in DESI.
+This page links to the URLs defining the bitmasks found in DESI files.
+
+Redshift fitting (Redrock) masks
```

Comment:
I would make these sub-headings, then Sphinx will automatically create links to the subheadings.